subsystem/cluster_test.go

```diff
@@ -262,8 +262,6 @@ func waitForHostState(ctx context.Context, clusterID strfmt.UUID, hostID strfmt.
 }
 
 func waitForClusterInstallationToStart(clusterID strfmt.UUID) {
-	waitForClusterState(context.Background(), clusterID, models.ClusterStatusPreparingForInstallation,
```

Comment:
Why not to add a validation of the status that return from the API?

Reply:
Added.